mozillaspeechlibrary/src/main/java/com/mozilla/speechlibrary/MozillaSpeechService.java

```diff
@@ -99,6 +113,31 @@ public void setLanguage(String language) {
         this.mNetworkSettings.mLanguage = language;
     }
 
+    public String getLanguageDir() {
+        return LocalSpeechRecognition.getLanguageDir(this.mNetworkSettings.mLanguage);
+    }
+
+    public void useDeepSpeech(boolean yesOrNo) {
+        this.useDeepSpeech = yesOrNo;
+    }
+
+    public String getModelPath() {
+        return this.mModelPath;
+    }
+
+    // This sets model's root path, not including the language
+    public void setModelPath(String aModelPath) {
+        this.mModelPath = aModelPath;
+    }
+
+    public boolean ensureModelInstalled() {
+        return LocalSpeechRecognition.ensureModelInstalled(this.getModelPath() + "/" + this.getLanguageDir());
+    }
+
+    public String getModelDownloadURL() {
```

Comment:
This is intended to provide access to "default" models. Packaging and hosting is open for debate.